cli/auth.go

```diff
@@ -161,7 +196,11 @@ func (c *viamClient) printAccessTokenAction(cCtx *cli.Context) error {
 		return err
 	}
 
-	fmt.Fprintln(cCtx.App.Writer, c.conf.Auth.AccessToken)
+	if token, ok := c.conf.Auth.(*token); ok {
+		fmt.Fprintln(cCtx.App.Writer, token.AccessToken)
+	} else {
+		return errors.New("not logged in as a user. cannot print access token")
```

Comment:
Might need a more informative error here. I don't think it will be obvious to users that they would need to logout and log back in with just `login` instead of `login api-key` in order to have an access token.